README.md

```diff
@@ -482,23 +482,23 @@ Quick example for #3. Today's date is 2020-01-18, a todo contains `2020-01-17 Te
 and TTDL is launched with relative dates enabled. If plugin `plug` does not exist, and relative it
```

Comment:
A minor, it my error, but could you include the fix into your MR:  remove `and relative` at the end of this line making it `and TTDL is launched with relative dates enabled. If plugin `plug` does not exist, it`.

Otherwise, the sentence looks weird.